examples/pip_install/WORKSPACE

```diff
@@ -36,7 +36,7 @@ pip_install(
     #environment = {"HTTP_PROXY": "http://my.proxy.fun/"},
 
     # Uses the default repository name "pip"
-    requirements = "//:requirements.txt",
+    requirements = "//:requirements.in",
```

Comment:
I'd still rather just lock these dependencies for linux to make CI pass. It's healthier for users to see examples with pinned transitive deps.

Reply:
Can you suggest a way to do that? The way these examples run in CI is unlike any other rule set but I have no problem making that change

Reply:
just what it says in `requirements.txt`: run `bazel run //:requirements.update` on linux

Reply:
Updated.